courses/migrations/0008_course_ids_backfilled.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9 on 2015-12-16 14:52
+from __future__ import unicode_literals
+
+import uuid
+
+from django.db import migrations, models
+
+def gen_uid(apps, schema_editor):
+    return uuid.uuid4().hex
```

Comment:
I'll run this when I'm done with reviewing the starcellbio PRs, but is returning a value here the correct thing to do? I thought you needed to grab the model from `apps.get_model` and adjust it that way (for example https://github.com/mitodl/lore/blob/master/learningresources/migrations/0015_backfill_curator_vocabularies.py)